packages/graphqlgen/src/tests/typescript/__snapshots__/basic.test.ts.snap

```diff
@@ -245,29 +245,29 @@ export namespace AddMemberPayloadResolvers {
     args: {},
     ctx: Context,
     info: GraphQLResolveInfo
-  ) => string | null | Promise<string | null>;
+  ) => string | null | undefined | Promise<string | null | undefined>;
```

Comment:
I'm not sure resolvers should change to accept `undefined` as a return value because it exposes the developer to making more mistakes such as forgetting to `return` something.